src/lib/y2partitioner/actions/delete_partition.rb

```diff
@@ -53,12 +44,42 @@ def delete
         UIState.instance.select_row(disk_device)
       end
 
-    private
+      # @see DeleteDevice#errors
+      def errors
+        errors = super + [implicit_partition_table_error]
+        errors.compact
+      end
+
+      # Error when the device contains an implicit partition table
+      #
+      # @return [String, nil] nil if the device does not contain an implicit
+      #   partition table.
+      def implicit_partition_table_error
+        return nil if !device.implicit_partition_table?
+
+        _("This partition cannot be deleted. Implicit partition table\n" \
+          "must have one partition.")
```

Comment:
Same than before, the "implicit partition table" is probably a non-sense outside storage-ng. So I would say something like "This is an implicit partition and cannot be deleted.".